components/Currency/CurrencyAmount/CurrencyAmount.tsx

```diff
@@ -1,19 +1,20 @@
 import React, { FC } from 'react';
 import styled from 'styled-components';
 
-import { formatCurrency, NumericValue, toBigNumber } from 'utils/formatters/number';
+import { formatCurrency } from 'utils/formatters/number';
 
 import { CurrencyKey } from 'constants/currency';
 
 import { ContainerRow } from '../common';
+import Wei, { wei } from '@synthetixio/wei';
 
 type CurrencyAmountProps = {
 	amountCurrencyKey: CurrencyKey;
-	amount: NumericValue;
+	amount: Wei | number | string;
```

Comment:
actually, it should be `WeiSource` (since that type is exported now). Wei source is convenient because you can still input numbers, and its no problem either way